civiform_config.example.sh

```diff
@@ -97,9 +97,9 @@ export STAGING_TI_NOTIFICATION_MAILING_LIST=""
 export STAGING_APPLICANT_NOTIFICATION_MAILING_LIST=""
 
 # REQUIRED
-# The custom domain name for this CiviForm deployment, not including the
-# protocol. E.g. "civiform.seattle.gov"
-export CUSTOM_HOSTNAME=""
```

Comment:
This should be moved to the azure section of the template, or fully removed from the azure configs

Reply:
Moved it to the azure section.